content/en/blog/_posts/2023-10-31-Gateway-API-GA/index.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: blog
+title: "Gateway API v1.0: GA Release"
+date: 2023-10-31T10:00:00-08:00
+slug: gateway-api-ga
+---
+
+***Authors:*** Shane Utt (Kong), Nick Young (Isovalent), Rob Scott (Google)
+
+We are pleased to announce the v1.0 release of [Gateway
```

Comment:
```suggestion
On behalf of Kubernetes SIG Network, we are pleased to announce the v1.0 release of [Gateway
```

otherwise, each “our” in the article infers it means you three